src/NetMQ/Msg.cs

```diff
@@ -225,15 +233,17 @@ public void InitPool(int size)
         /// </summary>
         /// <param name="data">the byte-array of data to assign to the Msg's Data property</param>
         /// <param name="size">the number of bytes that are in the data byte-array</param>
-        public void InitGC([NotNull] byte[] data, int size)
+        public void InitGC([NotNull] byte[] data, int size, int offset = 0)
```

Comment:
I think it is better to create a new constructor here, first this can break if you are not recompiling.
Also it would be better to keep .net structure of array,offset,length.